editors/code/package.json

```diff
@@ -219,6 +219,11 @@
                         }
                     }
                 },
+                "rust-analyzer.askBeforeDownload": {
```

Comment:
Let's group this into `rust-analyzer.updates`, as we might want to add `.chanel` soon as well? We should refactor old options to use the same grouping at some point, but it makes sense to design new options with more up-front planing that we use to have.